llvm/test/CodeGen/AArch64/GlobalISel/extract_subvector.ll

```diff
@@ -0,0 +1,53 @@
+; NOTE: Assertions have been autogenerated by utils/update_llc_test_checks.py UTC_ARGS: --version 5
+; RUN: llc < %s -mtriple aarch64 -mattr=+sve  | FileCheck %s
```

Comment:
non globalisel run line in the globalisel test? At least needs an explicit -global-isel=0

Reply:
I can move it one directory up. I couldn't find any existing tests with similar coverage.